apps/chat/src/components/Chat/TalkTo/TalkToSlider.tsx

```diff
@@ -161,6 +167,19 @@ const SliderModelsGroup = ({
             (model.reference === REPLAY_AS_IS_MODEL &&
               isReplayAsIsConversation(conversation));
 
+          if (model === SuggestedCard) {
+            return (
+              <div
+                className="flex size-full cursor-pointer flex-col items-center justify-center gap-2 rounded-md border border-primary hover:bg-layer-3"
+                onClick={onOpenMarketplaceTab}
+                key={SuggestedCard.id}
+              >
+                <h3>{t("Couln't find what you need?")}</h3>
```

Comment:
There is a typo here
```suggestion
                <h3>{t("Couldn't find what you need?")}</h3>
```